src/client/QXmppClient.h

```diff
@@ -180,6 +181,8 @@ class QXMPP_EXPORT QXmppClient : public QXmppLoggable
     bool isAuthenticated() const;
     bool isConnected() const;
 
+    void setIsAccountBeingDeleted(bool isAccountBeingDeleted);
```

Comment:
This is no nice way of handling that. A nice way would be something like a hook that is called that managers could handle.

But in general I don't think that either the user (using the Client) or managers should have to handle or should be able to handle stream errors.